launcher.py

```diff
@@ -0,0 +1,358 @@
+import argparse
+import atexit
+import json
+import os
+import platform
+import subprocess
+import sys
+import time
+import webbrowser
+from pathlib import Path
+
+CONFIG_FILE = os.path.join("config.json")
+venv_name = "virtual_env"  # For downloading python packages
+venv_dir = Path.cwd() / venv_name  # Create the path for the virtual environment
+
+
+def execute_command(command, cwd=None):
```

Comment:
For both these lines l14 and l17, as a general remark: let's try to reuse the existing verbs for these actions. For example to make something we've strictly used generate instead of create, and similarly to perform something we've currently used runCommand instead of execute. Please search for similar occurrences in general